### PR TITLE
Bump built to 0.2.2 to fix #917 build failure

### LINKED ISSUE #917: Fails to compile, "thread 'main' panicked at 'Box<Any>'" in built-0.2.0
Which operating system does the issue occur on?
Manjaro Linux, Ubuntu Linux 16.04
rust=1.22.1
cargo=0.23.0

If on linux, are you using X11 or Wayland?
X11

I've tried clearing out `~/.cargo` and re-cloning the alacritty repo; same result.

```
$ RUST_BACKTRACE=1 cargo build --release
   Compiling alacritty v0.1.0 (file:///home/jonathon/src/alacritty)
error: failed to run custom build command for `alacritty v0.1.0 (file:///home/jonathon/src/alacritty)`
process didn't exit successfully: `/home/jonathon/src/alacritty/target/release/build/alacritty-8bce53a622b9f242/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Box<Any>', /home/jonathon/.cargo/registry/src/github.com-1ecc6299db9ec823/built-0.2.0/src/lib.rs:339:18
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at /checkout/src/libstd/sys_common/backtrace.rs:69
   2: std::panicking::default_hook::{{closure}}
             at /checkout/src/libstd/sys_common/backtrace.rs:58
             at /checkout/src/libstd/panicking.rs:381
   3: std::panicking::default_hook
             at /checkout/src/libstd/panicking.rs:397
   4: std::panicking::rust_panic_with_hook
             at /checkout/src/libstd/panicking.rs:577
   5: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:538
   6: built::write_built_file_with_opts
             at /home/jonathon/.cargo/registry/src/github.com-1ecc6299db9ec823/built-0.2.0/src/lib.rs:339
             at /home/jonathon/.cargo/registry/src/github.com-1ecc6299db9ec823/built-0.2.0/src/lib.rs:807
   7: built::write_built_file
             at /home/jonathon/.cargo/registry/src/github.com-1ecc6299db9ec823/built-0.2.0/src/lib.rs:835
   8: build_script_build::main
             at ./build.rs:32
   9: __rust_maybe_catch_panic
             at /checkout/src/libpanic_unwind/lib.rs:99
  10: std::rt::lang_start
             at /checkout/src/libstd/panicking.rs:459
             at /checkout/src/libstd/panic.rs:361
             at /checkout/src/libstd/rt.rs:59
  11: __libc_start_main
  12: _start
```

This started a few commits ago, but:

```
$ git log -1
commit 8ff3c5d170abe47554f5d2a41ad35ddc451d254d (grafted, HEAD -> master, origin/master, origin/HEAD)
Author: Joe Wilm <joe@>
Date:   Mon Nov 20 09:30:57 2017 -0800

    Revert "Wait to display window until initialization is complete (#907)"
    
    This reverts commit a931d691a21a851ca4653cbab3542c600b72ff36.
    
    Please see https://github.com/jwilm/alacritty/pull/907#issuecomment-345666727
    for rationale w/ GIF.
    
    > now alacritty starts "in stages". First it paints some small
    > rectangle, and in a second in "boots" and fills up the available
    > space.
```

```
$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-pc-linux-gnu/7.2.0/lto-wrapper
Target: x86_64-pc-linux-gnu
Configured with: /build/gcc-multilib/src/gcc/configure --prefix=/usr --libdir=/usr/lib --libexecdir=/usr/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=https://bugs.archlinux.org/ --enable-languages=c,c++,ada,fortran,go,lto,objc,obj-c++ --enable-shared --enable-threads=posix --enable-libmpx --with-system-zlib --with-isl --enable-__cxa_atexit --disable-libunwind-exceptions --enable-clocale=gnu --disable-libstdcxx-pch --disable-libssp --enable-gnu-unique-object --enable-linker-build-id --enable-lto --enable-plugin --enable-install-libiberty --with-linker-hash-style=gnu --enable-gnu-indirect-function --enable-multilib --disable-werror --enable-checking=release --enable-default-pie --enable-default-ssp
Thread model: posix
gcc version 7.2.0 (GCC)
```

### PR DESCRIPTION
**Pull request by jonathonf**
See also https://github.com/lukaslueg/built/issues/7

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -48,7 +48,7 @@ nightly = []
 bench = []
 
 [build-dependencies]
-built = "0.2.0"
```
Comment (lukaslueg) [inline Cargo.toml]: Since this is an API-compatible change, "^0.2" should be enough and pick-up this and any further updates which are compatible.

Comment (jonathonf) [reply Cargo.toml]: I don't know enough about how the project likes to manage its `Cargo.toml` versioning so I'll defer this. Seems reasonable enough to me, though.

Comment (lukaslueg) [reply Cargo.toml]: I just bumped to 0.2.2, which updates some dependencies to more recent versions.

Comment (jonathonf) [reply Cargo.toml]: https://github.com/jwilm/alacritty/pull/932/commits/a397443886d9f860faee9518af74758eb75086fb

**Comment by jwilm** [comment]
Thanks for the PR! I've actually decided to remove the `built` crate for the time being. Rationale: continued support burden (both on and outside GitHub) due to build issues, and large extra dependencies.

**Pull request closed without merging**